pkg/manager/wrapper/bmh_mapper.go

```diff
@@ -27,3 +27,8 @@ func (m *bmhMapper) Map(obj client.Object) []reconcile.Request {
 	}
 	return []reconcile.Request{}
 }
+
+var bmhMapFunc = func(obj client.Object) []reconcile.Request {
```

Comment:
If the bmhMapper type isn't needed any more, let's just get rid of it and define the function we need directly.
Given that this never changes I'm not sure there's much point storing it as a var (rather than just defining a function) either.

Reply:
Done.